esignet-core/src/main/java/io/mosip/esignet/core/dto/ClientDetailCreateRequestV2.java

```diff
@@ -22,7 +23,7 @@
 public class ClientDetailCreateRequestV2 extends ClientDetailCreateRequest {
 
     @NotEmpty(message = ErrorConstants.INVALID_CLIENT_NAME)
-    private Map<@Size(message= ErrorConstants.INVALID_CLIENT_NAME_MAP_KEY, min=3, max=3) String,
+    private Map<@Size(message= ErrorConstants.INVALID_CLIENT_NAME_MAP_KEY, min=3, max=3) @ClientNameLang String,
```

Comment:
when we already have clientNameLang validator should be need @Size validator here?